### PR TITLE
Implement message signing

### PR DESCRIPTION
**Pull request by air1one**
For issue #10 .

### CODE REVIEW

File: spec/ark/crypto_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+require 'ark/util/crypto'
+
+describe Ark::Util::Crypto do
+  let(:message) { 'This is a message from PHP' }
```
Comment (faustbrian) [inline spec/ark/crypto_spec.rb]: but is it really from PHP? 👀

File: lib/ark/util/crypto.rb [hunk 2]
```diff
@@ -7,6 +7,10 @@ def self.get_key(secret)
         BTC::Key.new(:private_key => Digest::SHA256.digest(secret), :public_key_compressed => true)
       end
 
+      def self.get_pub_key(public_key)
```
Comment (faustbrian) [inline lib/ark/util/crypto.rb]: Please rename it to `get_public_key`, not gaining anything by using abbreviations.

File: spec/ark/crypto_spec.rb (hunk 1 above)
Comment (air1one) [reply spec/ark/crypto_spec.rb]: Hehe copy-pasted this 😅 Can change it to Ruby

File: lib/ark/util/crypto.rb (hunk 2 above)
Comment (air1one) [reply lib/ark/util/crypto.rb]: Right

**Pull request merged**